rancher/install-rancher.sh

```diff
@@ -0,0 +1,306 @@
+  displayName: KVM Falkenberg Large (16vCPU, 32GB RAM, 150GB Disk)
+  driver: glesys
+  engineInstallURL: https://releases.rancher.com/install-docker/19.03.sh
+  engineRegistryMirror: null
+  useInternalIpAddress: true
+---
+apiVersion: management.cattle.io/v3
+glesysConfig:
+  apiKey: $APIKEY
+  bandwidth: "100"
+  campaignCode: ""
+  cpu: "2"
+  dataCenter: Stockholm
+  memory: "4096"
+  platform: KVM
+  project: $PROJECT
+  rootPassword: ""
+  sshKeyPath: ""
+  storage: "20"
+  template: ubuntu-18-04
+  usernameKvm: docker-machine
+kind: NodeTemplate
+metadata:
+  annotations:
+    ownerBindingsCreated: "false"
+  name: glesys-kvm-stockholm-small
+  namespace: cattle-global-nt
+  labels:
+    Datacenter: Stockholm
+spec:
+  displayName: KVM Stockholm Small (2vCPU, 4GB RAM, 20GB Disk)
+  driver: glesys
+  engineInstallURL: https://releases.rancher.com/install-docker/19.03.sh
+  engineRegistryMirror: null
+  useInternalIpAddress: true
+---
+apiVersion: management.cattle.io/v3
+glesysConfig:
+  apiKey: $APIKEY
+  bandwidth: "100"
+  campaignCode: ""
+  cpu: "4"
+  dataCenter: Stockholm
+  memory: "8192"
+  platform: KVM
+  project: $PROJECT
+  rootPassword: ""
+  sshKeyPath: ""
+  storage: "60"
+  template: ubuntu-18-04
+  usernameKvm: docker-machine
+kind: NodeTemplate
+metadata:
+  annotations:
+    ownerBindingsCreated: "false"
+  name: glesys-kvm-stockholm-medium
+  namespace: cattle-global-nt
+  labels:
+    Datacenter: Stockholm
+spec:
+  displayName: KVM Stockholm Medium (4vCPU, 8GB RAM, 60GB Disk)
+  driver: glesys
+  engineInstallURL: https://releases.rancher.com/install-docker/19.03.sh
+  engineRegistryMirror: null
+  useInternalIpAddress: true
+---
+apiVersion: management.cattle.io/v3
+glesysConfig:
+  apiKey: $APIKEY
+  bandwidth: "100"
+  campaignCode: ""
+  cpu: "16"
+  dataCenter: Stockholm
+  memory: "32768"
+  platform: KVM
+  project: $PROJECT
+  rootPassword: ""
+  sshKeyPath: ""
+  storage: "150"
+  template: ubuntu-18-04
+  usernameKvm: docker-machine
+kind: NodeTemplate
+metadata:
+  annotations:
+    ownerBindingsCreated: "false"
+  name: glesys-kvm-stockholm-large
+  namespace: cattle-global-nt
+  labels:
+    Datacenter: Stockholm
+spec:
+  displayName: KVM Stockholm Large (16vCPU, 32GB RAM, 150GB Disk)
+  driver: glesys
+  engineInstallURL: https://releases.rancher.com/install-docker/19.03.sh
+  engineRegistryMirror: null
+  useInternalIpAddress: true
+EOF
+
+
+# Create SSH keys
+ssh-keygen -f /root/.ssh/id_rsa -P "" -q
+cat /root/.ssh/id_rsa.pub >> /root/.ssh/authorized_keys
+
+# Install Docker
+curl https://releases.rancher.com/install-docker/19.03.sh | sh
+
+### Download dependency
+
+# rke
+wget https://github.com/rancher/rke/releases/download/v1.1.2/rke_linux-amd64
+mv rke_linux-amd64 /usr/local/bin/rke
+chmod +x  /usr/local/bin/rke
+
+# kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
```

Comment:
Hopefully they have some guarantee on what that txt file will ever deliver.
Maybe quote it and leave legacy backticks?

`curl -LO https://storage.googleapis.com/kubernetes-release/release/"$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"/bin/linux/amd64/kubectl`